source/declarations.tex

```diff
@@ -6921,6 +6921,7 @@
 \end{codeblock}
 \end{example}
 \indextext{member!enumerator}%
+\begin{note}
 An enumerator declared in class scope can be referred to using the class
 member access operators (\tcode{::}, \tcode{.} (dot) and \tcode{->}
```

Comment:
I appreciate we might want a change here, but maybe we should start with the obvious stuff here: `::` is not a class member access operator and is not specified in [expr.ref]. Maybe [class.qual] is a better target for that case.

Reply:
How about:
> An enumerator declared in class scope can be referred to using the class
member access operators `.` (dot) and `->` (arrow) ([expr.ref]), or with the scope resolution operator `::`([class.qual]).